.github/workflows/pr-e2e.yml

```diff
@@ -48,13 +51,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Remove '${{ env.E2E_LABEL}}' label
+      - name: Remove '${{ env.E2E_LABELS_PASSED }}' label
         id: remove-label
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           git config --global --add safe.directory "$GITHUB_WORKSPACE"
-          gh pr edit ${{ needs.triage.outputs.pr_num }} --remove-label '${{ env.E2E_LABEL}}'
+          gh pr edit ${{ needs.triage.outputs.pr_num }} --remove-label '${{ env.E2E_LABELS_PASSED}}'
```

Comment:
Should we add here other labels like `not-run`?

Reply:
Good idea, but will see if we want this based on the main conversation